src/Toolkit/Collection.php

```diff
@@ -412,6 +412,11 @@ public function find(...$keys)
 	 */
 	public function findBy(string $attribute, $value)
 	{
+		// $value: cast object to string, e.g. UUID object
+		if (is_object($value) === true) {
```

Comment:
As Lukas posted in Discord, this one would be less prone to unexpected regressions. 

```suggestion
		if (is_a($value, 'Kirby\Uuid\Uuid') === true) {
```

Reply:
But then we should do that in Cms\Collection. And we need to get used to `instanceof`, not `is_a`.